spaces/domains.json

```
@@ -562,11 +562,12 @@
   "vote.gbox.space": "gboxspace.eth",
   "vote.guttercatgang.com": "gcgdao.eth",
   "vote.pagedao.org": "pagedao.eth",
-  "gov.fries.fund": "friesdao.eth",
+  "vote.starsharks.com": "starsharks.com"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.starsharks.com": "starsharks.com",
```